internal/rpc/rpc.go

```diff
@@ -60,8 +61,9 @@ func RunRpcServer(pk []byte, chainService chainservice.ChainService,
 		Logger()
 
 	if useDurableStore {
-		logger.Info().Msg("Initialising durable store...")
-		dataFolder := fmt.Sprintf("./data/nitro-service/%s", me.String())
+		dataFolder := filepath.Join(durableStoreFolder, me.String())
```

Comment:
Since we always use our address as a subfolder in `durableStoreFolder` we don't have to worry about making sure each go-nitro instance is given a unique `durableStoreFolder`. Multiple go-nitro instances can use the same `durableStoreFolder`, as long as they're each using a different SC address.